packages/core/src/plugins/ConsentPlugin.ts

```diff
@@ -151,6 +163,19 @@ export class ConsentPlugin extends Plugin {
       throw e;
     });
   }
+
+  public start() {
```

Comment:
when is this function getting called? didn't see it anyway except in unit test. maybe need to be invoked via sovran state changes?

Reply:
it's called by the user when they want to start processing queued events (could depend on when their CMP is initialized) and it's how swift/kotlin work. This way, we queue events until they want to be processed. Otherwise, events like Application Lifecycle events don't get stamped with consent preferences.

Reply:
described in the `readme` if you're curious!

Reply:
ah. I see. should've read the file. in that case, we may want to bound the queue to avoid OOM